portfolio/src/test/java/com/google/sps/CommentsTest.java

```diff
@@ -0,0 +1,26 @@
+package com.google.sps;
```

Comment:
Did you intend to include this in this PR?

Reply:
I did not. I have removed the test files.